Adding new groovy scripts for drupal and wordpress

vaibhavsood opened the pull request:
Hi Tianon,

I have created a couple of groovy scripts for drupal and wordpress, I have tested the scripts for a ppc64le machine but both these packages are php-based and hence cross-platform. Submitting these scripts to be included in your repo

Thanks,
Vaibhav

Commit:
Adding new groovy scripts for drupal and wordpress

tianon commented on dsl/docker-multiarch/images/wordpress.groovy:
```diff
@@ -0,0 +1,31 @@
+		wrappers { colorizeOutput() }
+		steps {
+			shell(multiarch.templateArgs(meta, ['dpkgArch']) + '''
+sed -i "s!^FROM !FROM $prefix/!" */{,*/}Dockerfile
```
The arg here should just be `*/Dockerfile` -- what's here will expand to `*/Dockerfile` and `*/*/Dockerfile`, and the second of these won't match anything since Wordpress doesn't have any sub-variants. :smile:

tianon commented on dsl/docker-multiarch/images/wordpress.groovy:
```diff
@@ -0,0 +1,31 @@
+sed -i "s!^FROM !FROM $prefix/!" */{,*/}Dockerfile
+
+docker build -t "$repo:apache" "apache"
+docker build -t "$repo:fpm" "fpm"
```
Mind dropping the unnecessary double quotes on these directory names and adding a `docker tag` line for latest?
ie:

``` bash
docker build -t "$repo:apache" apache
docker build -t "$repo:fpm" fpm
docker tag -f "$repo:apache" "$repo"
```

tianon commented on dsl/docker-multiarch/images/drupal.groovy:
```diff
@@ -0,0 +1,38 @@
+		wrappers { colorizeOutput() }
+		steps {
+			shell(multiarch.templateArgs(meta, ['dpkgArch']) + '''
+sed -i "s!^FROM !FROM $prefix/!" */{,*/}Dockerfile
```
Same note here -- no variants for Drupal yet so it's just `*/Dockerfile` for now. :smile:

tianon commented:
The style of these look really familiar! :heart: :+1:

I'm happy to take over from here if you'd like, but I want to make sure that you have the chance to amend the PR yourself if you want to do so. :smile:

We should probably also make sure both of these are listed in `dsl/docker-multiarch/docs.groovy` and `dsl/docker-multiarch/views.groovy` so they get docs and show up in my "images" tab. :innocent:

Commit:
Addressing review comments

Commit:
Addressing review comments

Commit:
Merge branch 'master' of https://github.com/vaibhavsood/jenkins-groovy

vaibhavsood commented:
Hi Tianon, thanks for your reply and review comments!! 

I have updated the PR addressing your comments. I have tried to make the scripts similar to the ones already in the repo (which is helping me understand the scripts better :) ) and have tested them on my jenkins setup

Thanks!
Vaibhav

tianon commented:
Nice, looks good!  Thanks!
